.github/workflows/main.yml

```diff
@@ -30,22 +30,23 @@ jobs:
     - name: Checkout OpenDDS
       uses: actions/checkout@v4
       with:
-        repository: 'objectcomputing/OpenDDS'
-        path: OpenDDS
+        repository: 'OpenDDS/OpenDDS'
+        path: opendds-3.30
```

Comment:
Why a hard-coded path like this?

Reply:
@iguessthislldo It seems to be another rule of Debian.  It was fixing this warning

dpkg-source: warning: source directory 'OpenDDS' is not <sourcepackage>-<upstreamversion> 'opendds-3.30'

And I assume it is to create the archive of the source code.

Reply:
Well, I wasn't yet their to put the version into a variable.